clustering/server/src/main/java/org/wildfly/clustering/server/dispatcher/LocalCommandDispatcher.java

```diff
@@ -114,6 +115,14 @@ public R call() throws Exception {
 
     @Override
     public void close() {
-        this.executor.shutdown();
+        PrivilegedAction<Void> closeAction = () -> {
+            this.executor.shutdown();
+            return null;
+        };
+        if (System.getSecurityManager() == null) {
```

Comment:
We should leverage the appropriate org.wildfly.security.manager.WildFlySecurityManager method here.